kotlin/src/main/com/looker/rtl/Constants.kt

```diff
@@ -30,12 +30,12 @@ class DelimArray<T> : Array<T>() {
 
 fun isTrue(value: String?) : Boolean {
     val low = value?.toLowerCase()
-    return low === "true" || low === "1" || low === "t" || low === "y" || low === "yes"
+    return low == "true" || low == "1" || low == "t" || low == "y" || low == "yes"
```

Comment:
Sorry, late to the party.  Why are we allowing/accepting loose boolean values?

Reply:
For implementation consistency with Python and Typescript's config processing